types/graphql-upload/package.json

```
@@ -1,6 +1,6 @@
 {
   "private": true,
   "dependencies": {
-    "graphql": "^16.2.0"
+    "graphql": "^15.3.0 || ^16.2.0"
```

Could you use the same version definition?
```suggestion
    "graphql": "0.13.1 - 16"
```